video_xblock/static/css/brightcove.css

```diff
@@ -0,0 +1,339 @@
+    content: "\f026";
+    padding: 7px 0 0 9px;
+}
+.video-js.vjs-fullscreen .vjs-fullscreen-control:before, .vjs-icon-fullscreen-exit:before {
+    content: "\f066";
+}
+.video-js .vjs-big-play-button:before, .video-js .vjs-play-control:before, .vjs-icon-play:before {
+    content: "\f04b";
+}
+.video-js .vjs-play-control.vjs-playing:before, .vjs-icon-pause:before {
+    content: "\f04c";
+}
+.video-js .vjs-remaining-time {
+    display: none;
+}
+.video-js .vjs-fullscreen-control {
+    position: absolute;
+    right: 0;
+    border-left: 1px dotted #4f595d;
+    width: 41px;
+}
+.video-js .vjs-volume-menu-button {
+    position: absolute;
+    right: 9em;
+    border-left: 1px dotted #4f595d;
+}
+.video-js .vjs-playback-rate {
+    position: absolute;
+    right: 41px;
+    border-left: 1px dotted #4f595d;
+    width: 42px;
+}
+.bc-player-default_default .vjs-menu-button-popup .vjs-menu .vjs-menu-content {
+    width: 41px;
+    left: 30px;
+    background: #171a1b;
+}
+.video-js .roller {
+    height: 5px;
+    width: 5px;
+    background-color: #CB598D;
+}
+.video-js .vjs-mouse-display:before, .video-js .vjs-play-progress:before, .video-js .vjs-volume-level:before, .vjs-icon-circle:before {
+    content: "\f0c8";
+}
+.video-js .vjs-volume-level:before {
+    font-size: 1.9em;
+}
+.video-js .vjs-play-progress:before {
+    top: 0;
+    font-size: .7em;
+}
+.video-js .vjs-progress-control:hover .vjs-progress-holder {
+    font-size: 2em;
+}
+.vjs-play-progress.vjs-slider-bar.vjs-keep-tooltips-inside:after {
+    content: '';
+    transform-origin: bottom left;
+    transition: all 0.2s;
+    transform: translateX(50%);
+    top: 0;
+    right: 0;
+    height: 100%;
+    padding: 0;
+    width: 5px;
+    border-radius: 4px;
+    background: #cb598d;
+    position: absolute;
+    visibility: visible;
+    display: block;
+}
+.vjs-mouse.bc-player-default_default .vjs-progress-control:hover .vjs-play-progress.vjs-slider-bar.vjs-keep-tooltips-inside:after {
+    width: 10px;
+}
+.video-js .vjs-time-control {
+    font-size: 13px !important;
+}
+.bc-player-default_default .vjs-time-control.vjs-current-time {
+    margin-left: 22px !important;
+}
+.vjs-playback-rate .vjs-playback-rate-value {
+    top: 4px !important;
+    font-weight: bold;
+}
+.bc-player-default_default .vjs-menu-button-popup .vjs-menu .vjs-menu-item {
+    font-weight: bold;
+}
+.bc-player-default_default .vjs-menu-button-popup .vjs-menu .vjs-menu-item:hover {
+    text-shadow: none;
+    color: inherit;
+    background: rgba(115,133,159,.5);
+    border-radius: 0;
+}
+/*
+  styles for Brightcove's theme 'default', if we change theme's name, we must
+  change class name.
+*/
+.bc-player-default_default .vjs-big-play-button:active,
+.bc-player-default_default .vjs-big-play-button:focus,
+.bc-player-default_default:active .vjs-big-play-button,
+.bc-player-default_default:hover .vjs-big-play-button {
+    background-color: rgba(115, 133, 159, .5) !important;
+}
```

Comment:
it makes sense to create our own theme, so it will produce more safe upgrades (any changes in default theme, will not affect our UI).